content/blog/2023/06 /02

```diff
@@ -0,0 +1,17 @@
+title: "The journey of Letica and development assessments - Research Call with Jonathan Reams"
+created: 2023-06-02
+categories: 
+  - "research-call"
+authors: 
+  - Rufuspollock
+
+>This call forms part of our monthly Research Call events. To get involved with this and our community calls please join the Life Itself [Whatsapp Chat](https://chat.whatsapp.com/JNJCTZugNQn)
```

Comment:
Space after `>`

Reply:
I'm not sure we need the call to action to be to join the chat exactly - especially since the call to action for the research calls is sign up on zoom